packages/feedback/src/core/sendFeedback.ts

```diff
@@ -48,18 +51,39 @@ export async function sendFeedbackRequest(
       event: baseEvent,
     });
 
-    if (!feedbackEvent) {
-      return resolvedSyncPromise({});
+    if (client.emit) {
+      client.emit('beforeSendFeedback', feedbackEvent, { includeReplay: Boolean(includeReplay) });
     }
 
-    client.emit('beforeSendFeedback', feedbackEvent, { includeReplay: Boolean(includeReplay) });
+    try {
+      const response = await transport.send(
+        createEventEnvelope(feedbackEvent, dsn, client.getOptions()._metadata, client.getOptions().tunnel),
+      );
+
+      if (attachments && attachments.length) {
+        // TODO: https://docs.sentry.io/platforms/javascript/enriching-events/attachments/
+        await transport.send(
+          createAttachmentEnvelope(
+            feedbackEvent,
+            attachments,
+            dsn,
+            client.getOptions()._metadata,
+            client.getOptions().tunnel,
+          ),
+        );
+      }
 
-    const envelope = createEventEnvelope(feedbackEvent, dsn, client.getOptions()._metadata, client.getOptions().tunnel);
+      // TODO (v8): we can remove this guard once transport.send's type signature doesn't include void anymore
+      if (!response) {
+        throw new Error('Unable to send Feedback');
+      }
 
-    let response: TransportMakeRequestResponse;
+      // Require valid status codes, otherwise can assume feedback was not sent successfully
+      if (typeof response.statusCode === 'number' && (response.statusCode < 200 || response.statusCode >= 300)) {
+        throw new Error('Unable to send Feedback. Invalid response from server.');
```

Comment:
While we are at it, or in a follow up, we could check for statusCode === 0 and show a message like:

```
Unable to send Feedback. This is because of network issues, or because you are using an ad-blocker.
```

Or something along these lines...!